apps/emqx/src/emqx_broker.erl

```diff
@@ -72,12 +78,15 @@
     code_change/3
 ]).
 
--import(emqx_utils_ets, [lookup_value/2, lookup_value/3]).
+%% Tests-only
+-export([
```

Comment:
-ifdef(TEST)?

Reply:
Then we would have to guard both the export attribute and every function in the implementation chain because there would be warnings when compiling under non-test profile, and this feels a bit too ugly. 🫠

Reply:
`nowarn_unused_function`?